subiquity/server/controllers/install.py

```diff
@@ -339,6 +339,13 @@ async def postinstall(self, *, context):
                 ubuntu_drivers = self.app.controllers.Drivers.ubuntu_drivers
                 await ubuntu_drivers.install_drivers(root_dir=self.tpath(),
                                                      context=child)
+        if self.model.codecs.do_install:
+            # NOTE currently, ubuntu-restricted-addons is an empty package that
```

Comment:
Does this imply that we need curtin changes?

Reply:
I tried with a kinetic server ISO and apt installed the recommended packages so I think there is no need for an immediate change in curtin. If we were to build an image with `APT::Install-Recommends "false"`, the codecs would not get installed.